torch/_inductor/compile_worker/subproc_pool.py

```diff
@@ -1,4 +1,5 @@
-# mypy: allow-untyped-defs
+from __future__ import annotations
```

Comment:
You don't need this `__future__` at all, do you?

Reply:
I think you're right - it was from a transient change I had. But AFAIK it also causes no harm...

Reply:
I prefer it disabled as it prevents some of the linters from picking up errors in typehints (ie unimported names etc), since it stringifies everything.